### ISSUE TITLE
Users should be reminded how to change system settings to receive alerts from DailyDozen

### ISSUE THREAD

**Issue by sjl872964789** [nutritionfactsorg/daily-dozen-android#139]
First of all, thank you for developing this amazing application. Recently, I am trying to use daily reminders, but I always don't receive alarm reminders. I can't receive notifications until I find that I have to turn on self startup, notify and other permissions. 
Moreover, on my other phone, I still don't know how to set it up so that I can receive notifications, as if it was blocked by Android system. Can you guide the user to open the corresponding system settings when the user attempts to use this function. Attach the recording of mobile phone successfully notified after my attempt. 
[MI.zip](https://github.com/nutritionfactsorg/daily-dozen-android/files/3991143/MI.zip)

OS version: [Android 8.1.0]
App version: [17]

**Comment by slavick** [comment]
Closing this as it is out of scope for this app. Uninstalling and reinstalling the app should reset any app-specific notification settings to default.
